walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -42,7 +41,10 @@
   private static final int TIME_0930AM = TimeRange.getTimeInMinutes(9, 30);
   private static final int TIME_1000AM = TimeRange.getTimeInMinutes(10, 0);
   private static final int TIME_1100AM = TimeRange.getTimeInMinutes(11, 00);
+  private static final int TIME_1130AM = TimeRange.getTimeInMinutes(11, 30);
+  private static final int TIME_1200 = TimeRange.getTimeInMinutes(12, 00);
```

Comment:
nit - add PM here to be consistent with the other times.
```suggestion
  private static final int TIME_1200PM = TimeRange.getTimeInMinutes(12, 00);
```